app/src/main/java/net/squanchy/schedule/view/ScheduleViewPagerAdapter.kt

```diff
@@ -45,12 +46,12 @@ class ScheduleViewPagerAdapter(context: Context) : ViewPagerAdapter<ScheduleDayP
 
     override fun getPageTitle(position: Int): CharSequence? {
         val date = pages[position].date
-        return date.toString(TITLE_FORMAT_TEMPLATE).toUpperCase(Locale.getDefault())
+        return date.format(TITLE_FORMAT).toUpperCase(Locale.getDefault())
     }
 
     override fun isViewFromObject(view: View, anObject: Any) = view === anObject
 
     companion object {
-        private const val TITLE_FORMAT_TEMPLATE = "EEE d"
+        private val TITLE_FORMAT = DateTimeFormatter.ofPattern("EEE d")
```

Comment:
I'm not sure about this, it's used through the app quite often, but I didn't extract it (yet), how could we name it?

Reply:
Maybe we can put this into dagger as a Qualified binding and inject it wherever we need it

Reply:
b26dc37 for now I just extracted a function